modules/enterprise-management/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/model/Account.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2023.
+ * (C) Copyright IBM Corp. 2024.
```

Comment:
In the future, when a re-gen yields a new file that differs only in the copyright year, you can safely revert this change and avoid including it in your commit.